modules/libcom/RTEMS/posix/rtems_config.c

```diff
@@ -0,0 +1,164 @@
+/*************************************************************************\
+* Copyright (c) 2002 The University of Saskatchewan
+* EPICS BASE Versions 3.13.7
+* and higher are distributed subject to a Software License Agreement found
+* in file LICENSE that is included with this distribution. 
+\*************************************************************************/
+/*
+ * RTEMS configuration for EPICS
+ *      Author: W. Eric Norum
+ *              norume@aps.anl.gov
+ *              (630) 252-4793
+ */
+
+#include <rtems.h>
+
+/*
+ ***********************************************************************
+ *                         RTEMS CONFIGURATION                         *
+ ***********************************************************************
+ */
+
+extern void *POSIX_Init(void *argument);
+
+#define CONFIGURE_POSIX_INIT_THREAD_TABLE
+#define CONFIGURE_POSIX_INIT_THREAD_ENTRY_POINT POSIX_Init
+#define CONFIGURE_POSIX_INIT_THREAD_STACK_SIZE  (64*1024)
+
+#define CONFIGURE_MAXIMUM_PERIODS 	5
+#define CONFIGURE_MICROSECONDS_PER_TICK 10000
+#define CONFIGURE_MALLOC_STATISTICS     1
+/* MINIMUM_STACK_SIZE == 8K */
+#define CONFIGURE_EXTRA_TASK_STACKS         (4000 * RTEMS_MINIMUM_STACK_SIZE)
+
+#if __RTEMS_MAJOR__ > 4
+#define CONFIGURE_FILESYSTEM_DEVFS
+#define CONFIGURE_FILESYSTEM_TFTPFS
+#endif
+#define CONFIGURE_FILESYSTEM_NFS
+#define CONFIGURE_FILESYSTEM_IMFS
+
+#ifndef RTEMS_LEGACY_STACK
+#define CONFIGURE_USE_IMFS_AS_BASE_FILESYSTEM
+/*
+ * Configure LibBSD.
+ */
+//#define RTEMS_BSD_CONFIG_NET_PF_UNIX
+//#define RTEMS_BSD_CONFIG_NET_IF_BRIDGE
+//#define RTEMS_BSD_CONFIG_NET_IF_LAGG
+//#define RTEMS_BSD_CONFIG_NET_IF_VLAN
+#define RTEMS_BSD_CONFIG_BSP_CONFIG
+#define RTEMS_BSD_CONFIG_INIT
+#include <machine/rtems-bsd-config.h>
+#endif // not LEGACY_STACK
+
+/*
+ * Configure RTEMS.
+ */
+#define CONFIGURE_APPLICATION_NEEDS_CLOCK_DRIVER
+#define CONFIGURE_APPLICATION_NEEDS_CONSOLE_DRIVER
+#define CONFIGURE_APPLICATION_NEEDS_STUB_DRIVER
+#define CONFIGURE_APPLICATION_NEEDS_ZERO_DRIVER
+
+#define CONFIGURE_MAXIMUM_FILE_DESCRIPTORS 150
+#define CONFIGURE_IMFS_ENABLE_MKFIFO    2
+
+#define CONFIGURE_MAXIMUM_NFS_MOUNTS 		3
+#define CONFIGURE_MAXIMUM_USER_EXTENSIONS 	5
+
+#define CONFIGURE_UNLIMITED_ALLOCATION_SIZE 32
+#define CONFIGURE_UNLIMITED_OBJECTS
+#define CONFIGURE_UNIFIED_WORK_AREAS
+
+#define CONFIGURE_STACK_CHECKER_ENABLED
+
+#define CONFIGURE_APPLICATION_NEEDS_LIBBLOCK
+#define CONFIGURE_BDBUF_BUFFER_MAX_SIZE (64 * 1024)
+#define CONFIGURE_BDBUF_MAX_READ_AHEAD_BLOCKS 4
+#define CONFIGURE_BDBUF_CACHE_MEMORY_SIZE (1 * 1024 * 1024)
+
+/* we are using POSIX_INIT needed by V4
+#define CONFIGURE_RTEMS_INIT_TASKS_TABLE
+#define CONFIGURE_INIT_TASK_STACK_SIZE (32 * 1024)
+#define CONFIGURE_INIT_TASK_INITIAL_MODES RTEMS_DEFAULT_MODES
+#define CONFIGURE_INIT_TASK_ATTRIBUTES RTEMS_FLOATING_POINT
+#endif
+*/
+
+//#define RTEMS_PCI_CONFIG_LIB
+//#define CONFIGURE_PCI_LIB PCI_LIB_AUTO
+
+#ifndef RTEMS_LEGACY_STACK
```

Comment:
Should this be `__RTEMS_MAJOR__ > 4`?

Reply:
> Should this be `__RTEMS_MAJOR__ > 4`?

I think so.  Changed to check rtems version instead.